cdk/lib/support-reminders.ts

```diff
@@ -96,16 +156,16 @@ export class SupportReminders extends GuStack {
 					httpMethod: "POST",
 					lambda: reactivateRecurringReminderLambda,
 				},
-				{
-					path: "/create/recurring",
-					httpMethod: "POST",
-					lambda: createRemindersSignupLambda,
-				},
-				{
-					path: "/create/one-off",
-					httpMethod: "POST",
-					lambda: createRemindersSignupLambda,
-				},
+				// {
+				// 	path: "/create/recurring",
+				// 	httpMethod: "POST",
+				// 	lambda: createRemindersSignupLambda,
+				// },
+				// {
+				// 	path: "/create/one-off",
+				// 	httpMethod: "POST",
+				// 	lambda: createRemindersSignupLambda,
+				// },
```

Comment:
Do we need these commented out items?

Reply:
Oh sorry, we don't need that.I will remove those lines.